Assets/Scripts/Gameplay/GameplayObjects/Character/ClientCharacter.cs

```diff
@@ -140,9 +140,9 @@ public override void OnNetworkSpawn()
             {
                 name = "AvatarGraphics" + m_ServerCharacter.OwnerClientId;
 
-                if (m_ServerCharacter.TryGetComponent(out ClientAvatarGuidHandler clientAvatarGuidHandler))
+                if (m_ServerCharacter.TryGetComponent(out ClientPlayerAvatarNetworkAnimator characterNetworkAnimator))
```

Comment:
Minor thing...like super-minor...
If (in the event a user makes changes to a prefab) it cannot get the component...should it log an error/warning?
Does any other code try to access `m_ClientVisualsAnimator`?

The super nit-picky aspect is using TryGetComponent (as opposed to just GetComponent) without any handling of "what happens when the TryGetComponent fails". If there is no handling, then it might be better to just use GetComponent and allow the exception if something tries to use `m_ClientVisualsAnimator` or add an else with a message complaining about the component not being found.

Wouldn't block the PR for that...but figured worth pointing out the logic/usage conundrum/scenario.

Reply:
Thanks for the feedback!
We should definitely return a warning here if that's the case.

I've created [this task](https://jira.unity3d.com/browse/MTT-8378) to keep track of this work and include it in a future release!